src/main/kotlin/net/maizegenetics/phgv2/cli/CreateRanges.kt

```diff
@@ -3,14 +3,22 @@ package net.maizegenetics.phgv2.cli
 import biokotlin.featureTree.Feature
 import biokotlin.featureTree.Genome
 import biokotlin.featureTree.Strand
+import biokotlin.seq.NucSeq
+import biokotlin.seqIO.NucSeqIO
 import com.github.ajalt.clikt.core.CliktCommand
 import com.github.ajalt.clikt.parameters.options.default
+import com.github.ajalt.clikt.parameters.options.flag
 import com.github.ajalt.clikt.parameters.options.option
 import com.github.ajalt.clikt.parameters.options.validate
 import com.github.ajalt.clikt.parameters.types.choice
 import com.github.ajalt.clikt.parameters.types.int
 import java.io.File
 
+/**
+ * Data class to hold the information needed to output a BedRecord.
+ */
+data class BedRecord(val contig: String, val start : Int, val end: Int, val name: String, val score : Int, val strand: String )
```

Comment:
This might be used by other things?

Reply:
It could be, but likely will not.  I just needed a simple object to hold things in while I fill in the intergenic regions.